progetto_esa.tex

```diff
@@ -49,7 +48,7 @@ \section{What is build root}
 This would show the simplicity and the power of buildroot at the same time.
 
 \section{Basic Configuration}
-The first thing to do to work with build root is to create a configuration file according to your specific hardware and needs.
+The first thing to do to work with buildroot is to create a configuration file according to your specific hardware and needs.
```

Comment:
or to use one of the multiple ready available configurations if you have a supported board.